app/statistics/server/functions/get.js

```diff
@@ -129,16 +175,57 @@ statistics.get = function _getStatistics() {
 	};
 
 	statistics.uploadsTotal = Uploads.find().count();
-	const [result] = Promise.await(Uploads.model.rawCollection().aggregate([{ $group: { _id: 'total', total: { $sum: '$size' } } }]).toArray());
+	const [result] = Promise.await(
+		Uploads.model
+			.rawCollection()
+			.aggregate([{ $group: { _id: 'total', total: { $sum: '$size' } } }])
+			.toArray()
+	);
 	statistics.uploadsTotalSize = result ? result.total : 0;
 
 	statistics.migration = Migrations._getControl();
-	statistics.instanceCount = InstanceStatus.getCollection().find({ _updatedAt: { $gt: new Date(Date.now() - process.uptime() * 1000 - 2000) } }).count();
+	statistics.instanceCount = InstanceStatus.getCollection()
+		.find({
+			_updatedAt: { $gt: new Date(Date.now() - process.uptime() * 1000 - 2000) },
+		})
+		.count();
+
+	const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
 
-	const { oplogEnabled, mongoVersion, mongoStorageEngine } = getMongoInfo();
-	statistics.oplogEnabled = oplogEnabled;
-	statistics.mongoVersion = mongoVersion;
-	statistics.mongoStorageEngine = mongoStorageEngine;
+	if (mongo._oplogHandle && mongo._oplogHandle.onOplogEntry) {
```

Comment:
These changes here we'll definitely want whats in develop or will re-break some some issues

Reply:
That was not supposed to be here, act like it never happened 😆